tests/testing_data/data_config.json

```diff
@@ -56,6 +56,13 @@
             "hash_val": "eb4f1e596ca85aadaefc359d409fb9a3e27d733e6def04b996953b7c54bc26d4"
         }
     },
+    "videos": {
+        "endovis": {
+            "url": "https://github.com/rijobro/real_time_seg/raw/main/example_data/EndoVis2017/d1_im.mp4",
```

Comment:
this test case should be moved to https://github.com/Project-MONAI/MONAI-extra-test-data/releases/tag/0.8.1

currently it's giving:
```
[2022-08-08T09:56:01.566Z] ======================================================================
[2022-08-08T09:56:01.566Z] ERROR: test_dataset (tests.test_video_datasets.TestVideoFileDataset)
[2022-08-08T09:56:01.566Z] ----------------------------------------------------------------------
[2022-08-08T09:56:01.566Z] Traceback (most recent call last):
[2022-08-08T09:56:01.566Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_video_datasets.py", line 116, in test_dataset
[2022-08-08T09:56:01.566Z]     super().test_dataset(known_num_frames, known_fps)
[2022-08-08T09:56:01.566Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_video_datasets.py", line 69, in test_dataset
[2022-08-08T09:56:01.566Z]     ds = self.get_ds(max_num_frames=max_num_frames)
[2022-08-08T09:56:01.566Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_video_datasets.py", line 39, in get_ds
[2022-08-08T09:56:01.566Z]     return self.ds(video_source=self.video_source, transform=TRANSFORMS, *args, **kwargs)  # type: ignore
[2022-08-08T09:56:01.566Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/data/video_dataset.py", line 125, in __init__
[2022-08-08T09:56:01.566Z]     VideoDataset.__init__(self, *args, **kwargs)
[2022-08-08T09:56:01.566Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/data/video_dataset.py", line 72, in __init__
[2022-08-08T09:56:01.566Z]     self.cap = self.open_video(video_source)
[2022-08-08T09:56:01.566Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/data/video_dataset.py", line 92, in open_video
[2022-08-08T09:56:01.566Z]     raise RuntimeError(f"Failed to open video: {video_source}")
[2022-08-08T09:56:01.566Z] RuntimeError: Failed to open video: /tmp/tmp5sbe94z4/endo.mp4
```